Dockerfile

```diff
@@ -6,7 +6,8 @@ COPY package-lock.json .
 
 RUN npm install
 
-COPY index.js /home/balta/Documentos/Máster/Asignaturas/FIS/Proyecto/teams/teamsAPI/v1/
```

Comment:
We' ve got some problems with the copy of the files containig the implementation of the microservices' s apis

Reply:
Resolved with the following two lines

Reply:
Resolved with the following two lines